web/src/app/layout.js

```diff
@@ -3,30 +3,27 @@
  * This software is licensed under the Apache License version 2.
  */
 
-import '@/lib/styles/globals.css'
+import '@/src/lib/styles/globals.css'
```

Comment:
There's no need to modify all similar lines.
You can set the `compilerOptions` property `"@/*": ["./src/*"]` in the `tsconfig.json`.